101-servicebus-namespace/README.md

```diff
@@ -0,0 +1,7 @@
+# Create a Service Bus Namespace
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-servicebus-namespace%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template creates a Service Bus Namespace.
```

Comment:
Can you add some links where to find the Service Bus documentation and more info about Service Bus here?